test/programmatic_model_creation/component_based_model_creation.jl

```diff
@@ -21,6 +27,7 @@ let
     specs = [m, P, R]
     pars = [α₀, α, K, n, δ, β, μ]
     @named rs = ReactionSystem(rxs, t, specs, pars)
+    rs = complete(rs)
```

Comment:
This is one of those test files that have really long test blocks where it is hard to see what exactly is being tested. Long-term it would be nice to split these up. But lets fix v14 first.